integrations/wp-importer/wp-import.php

```diff
@@ -36,16 +36,15 @@ public function process_terms() {
 
 		parent::process_terms();
 
-		// Update the languages list if needed
 		// First reset the core terms cache as WordPress Importer calls wp_suspend_cache_invalidation( true );
 		wp_cache_set( 'last_changed', microtime(), 'terms' );
-		PLL()->model->clean_languages_cache();
 
-		if ( ( $options = get_option( 'polylang' ) ) && empty( $options['default_lang'] ) && ( $languages = PLL()->model->get_languages_list() ) ) {
-			// Assign the default language if importer created the first language
+		// Assign the default language in case the importer created the first language.
+		if ( PLL()->options && empty( PLL()->options['default_lang'] ) ) {
```

Comment:
Should we test for `PLL()->options`? I mean, why not do this directly:
```suggestion
		if ( empty( PLL()->options['default_lang'] ) ) {
```